test/makeTravisReport.m

```diff
@@ -61,13 +276,91 @@ function displaySummaryTable(stream, status)
             end
         end
         if isempty(summary)
-            summary = 'OK';
+            passOrFail = symbol.pass;
+        else
+            passOrFail = symbol.fail;
         end
         summary = strtrim(summary);
     end
-    functionName = strjust(sprintf('%25s', oneStatus.function), 'left');
+    row = { gfmCode(sprintf('%s(%d)', testSuite, testNumber)), ...
+            gfmCode(oneStatus.function), ...
+            passOrFail, ...
+            summary};
+end
+% ==============================================================================
+function displayTestSummary(stream, S)
+    % display a table of # of failed/passed/skipped tests vs (un)reliable
+    
+    % compute number of cases per category
+    reliableSummary   = cellfun(@numel, {S.passR, S.failR, S.skipR});
+    unreliableSummary = cellfun(@numel, {S.passU, S.failU, S.skipU});
+    
+    % make summary table + calculate totals
+    summary = [  reliableSummary                 numel(S.reliable);
+               unreliableSummary                 numel(S.unreliable);
+               reliableSummary+unreliableSummary numel(S.all)];
+           
+    % put results into cell array with proper layout
+    summary = arrayfun(@(v) sprintf('%d',v), summary, 'UniformOutput', false);
+    table = repmat({''}, 3, 5);
+    header = {'','Pass','Fail','Skip','Total'};
+    table(:,1) = {'Reliable','Unreliable','Total'};
```

Comment:
Should we change this to "unreliable, reliable, total" (different order), as it used to be?